### PR TITLE
Page body

### LINKED ISSUE #18: Practice Content Body
As Nova, I need consistency in the content from one practice to the next so that I feel a seamless experience with trusted resources that I will want to come back to and share.

**ACCEPTANCE CRITERIA** 

**Given** I am on any Practice page
**When** I view the content
**Then** I see the main body sections broken into What is... Why do... and How to...
**And** I the practice title is included in each section name.

**NOTES**
- Full practice page: https://zpl.io/amAnrdr
- Body copy only for this story - the thought is to auto generate the Practice Title after each section of the content
- See Media Gallery story https://github.com/openpracticelibrary/opl-fe/issues/19
- See Resources we love story https://github.com/openpracticelibrary/opl-fe/issues/20

### LINKED ISSUE #20: Resources we love
As Nova, I need a to see other resources about a particular practice so I feel confident in my search and can easily access curated content.

**ACCEPTANCE CRITERIA** 

**Given** I scroll to Resources section of any Practice page
**When** I see a list of Resources indicated by icon type
**Then** I can click on any to view a link, download a template or go to a podcast.

**NOTES**
- Full practice page: https://zpl.io/amAnrdr
- Needs designs for the interaction of adding a resource (may be a form for MVP)
- Add resource story https://github.com/openpracticelibrary/opl-fe/issues/28

### PR DESCRIPTION
**Pull request by Venus-Izadi**
resolves #18 and resolves #20

### CODE REVIEW

File: src/assets/icons/PodcastIcon.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+function PodcaseIcon() {
```
Comment (shaheinm) [inline src/assets/icons/PodcastIcon.js:3]: Spelling fix: `PodcastIcon`

File: src/components/practicePage/PageBody/MediaGallery.js [hunk 2]
```diff
@@ -15,30 +15,47 @@ const useStyles = makeStyles((theme) => ({
...
+    display: "flex",
+    justifyContent: "center",
+    margin: theme.spacing(1),
+    padding: theme.spacing(1),
   },
 }));
 
+const testImages = [
+  {
+    original: 'https://picsum.photos/id/1018/1000/600/',
```
Comment (shaheinm) [inline src/components/practicePage/PageBody/MediaGallery.js:28]: I'm not convinced a demo would be more impressive with these stock photos than the actual open practice library ones, even if they are _better_ photos.

File: src/graphql/client.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import ApolloClient from 'apollo-boost';
 // TODO: Set URI attribute to env var
 const client = new ApolloClient({
   //uri: 'http://localhost:4000'
-  uri: 'http://opl-cms-opl-dev.apps.s43.core.rht-labs.com/graphql'
+  uri: 'http://opl-content-api-opl-dev.apps.s44.core.rht-labs.com/graphql'
```
Comment (shaheinm) [inline src/graphql/client.js:6]: We should go ahead and remove the hard coding. I think I did so already in the branch for CI/CD (if I didn't I'm going to). You should use a local env file with `REACT_APP_GRAPHQL_ENDPOINT=http://opl-content-api-opl-dev.apps.s44.core.rht-labs.com/graphql` instead.

File: src/assets/icons/PodcastIcon.js (hunk 1 above)
Comment (Venus-Izadi) [reply src/assets/icons/PodcastIcon.js:3]: It's been fixed

File: src/components/practicePage/PageBody/MediaGallery.js (hunk 2 above)
Comment (Venus-Izadi) [reply src/components/practicePage/PageBody/MediaGallery.js:28]: That story for Media Gallery won't be finish by tomorrow. It's a bigger story than just adding a library.
The reason I put these nice photos was that they have the same sizing and resolution, and I don't need to spend time on fixing the Grid for the photo sizing or cutting photos or adding black areas for smaller photos. Or any other strategies!
I can hide this section completely for demo if you think photos are not appropriate.

File: src/graphql/client.js (hunk 3 above)
Comment (Venus-Izadi) [reply src/graphql/client.js:6]: Good idea. Working on it.

File: src/assets/icons/PodcastIcon.js (hunk 1 above)
Comment (shaheinm) [reply src/assets/icons/PodcastIcon.js]: Don't push anything, I already merged master into the branch

File: src/components/practicePage/PageBody/MediaGallery.js (hunk 2 above)
Comment (shaheinm) [reply src/components/practicePage/PageBody/MediaGallery.js]: That makes sense, leave them in. And yeah, it's definitely at least a 5 point story :wink:

**Pull request merged**